### DISCUSSION TITLE
can't push to origin

### DISCUSSION THREAD

**Discussion by homa8511** [evroon/bracket#692] [Q&A]
Github returns a 403 unauthorized when I want to push my changes to a new branch or an existing branch in this repository.

How can I provide my translation (German)?

**Comment by evroon** [comment]
Thanks for helping with translations. 
You can do two things: 
- Fork this repo and push the changes to your fork. Then you can create a PR from the branch you pushed changes to, to this repo.
- I think it should also be possible to enter the translations here: https://crowdin.com/project/bracket but I haven't tried it as non-admin yet. It should sync with GitHub.
